service-01/PUS-1-10.asn1

```diff
@@ -0,0 +1,36 @@
+--!
+--! Copyright (C) 2017 N7 Mobile sp. z o. o.
+--! Contact: http://n7mobile.pl/Space
+--!
+--! This file is part of ASN.1/ACN PUS-C Components Library.
+--!
+--! Library was developed under a programme and funded by
+--! European Space Agency.
+--!
+--! This Library is free software: you can redistribute it and/or modify
+--! it under the terms of the GNU General Public License as published by
+--! the Free Software Foundation, either version 3 of the License, or
+--! (at your option) any later version.
+--!
+--! This Library is distributed in the hope that it will be useful,
+--! but WITHOUT ANY WARRANTY; without even the implied warranty of
+--! MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+--! GNU General Public License for more details.
+--!
+--! You should have received a copy of the GNU General Public License
+--! along with this program.  If not, see <http://www.gnu.org/licenses/>.
+--!
+
+PUS-1-10 DEFINITIONS AUTOMATIC TAGS ::= BEGIN
+EXPORTS ALL;
+IMPORTS
+    Request-ID FROM Request-ID
+    FailureNotice FROM FailureNotice;
+
+TM-1-10-FailedRoutingVerificatrionReport ::= SEQUENCE
+{
+    request-ID Request-ID,
+    failureNotice FailureNotice
```

Comment:
IMHO:
module "FailuresNotices"
Type "RoutingFailureNotice", "AcceptanceFailureNotice" etc - different notices for different kind of errors

Reply:
Overlooked this, will be fixed

Reply:
Fixed as suggested, but maybe it's better idea to separate each FailureNotice and ErrorCodes than keep all types in one module? Will be much more readable and easier for user to specify each ErrorCodes

Reply:
is any type used in more than one module? maybe each could just go into proper PUS-1-x module? I do not want to have too many modules - it's not easy to manipulate them

Reply:
After the changes every module uses it's own type for errors